pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+		return err
+	}
+
+	if cluster.Spec.Stork != nil && cluster.Spec.Stork.Enabled {
+		if err := validatePodsByName(cluster.Namespace, "stork", timeout, interval); err != nil {
```

Comment:
We can probably just use ValidateDeployment from sched-ops. It verifies that the number of pod is the same as given in the deployment. That way we are verifying the deployment is present and its pods are running.